src/fsharp/NameResolution.fs

```diff
@@ -3078,54 +3077,71 @@ let rec ResolvePartialLongIdentInModuleOrNamespace (ncenv: NameResolver) nenv is
     
     let tycons = 
         mty.TypeDefinitions
-        |> List.filter (fun tcref -> not (tcref.LogicalName.Contains(",")))
-        |> List.filter (fun tycon -> not (IsTyconUnseen ad g ncenv.amap m (modref.NestedTyconRef tycon)))
+        |> List.filter (fun tcref -> 
+            not (tcref.LogicalName.Contains(",")) &&
```

Comment:
There is one sample of "operator at the beginning" https://github.com/microsoft/visualfsharp/blob/f95ad73110d198bc75726962ade3edeeaaa27117/src/fsharp/tast.fs#L4441